README.md

```diff
@@ -19,17 +19,21 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Infrastructure and processes we build and/or support
 
+* [Forum](https://forum.ansible.com/): Official Ansible Discource forum.
 * [Meetings](https://github.com/ansible/community/tree/main/meetings): Matrix/IRC meeting-related infrastructure, including the meeting bot and space for logs; organization of Ansible Contributor Summits.
 * [Bullhorn](https://github.com/ansible/community/wiki/News#the-bullhorn): Ansible contributor newsletter.
 * [Collection Bot](https://github.com/ansible-community/collection_bot): Automation of GitHub issue/pull request processing for ansible collections.
 
 ## Groups
 
-* [All Ansible Working Groups](https://github.com/ansible/community/wiki)
+* Ansible Working Groups are now present in two forms:
+  * [On the Forum](https://forum.ansible.com/g) (recommended)
+  * [As wiki pages](https://github.com/ansible/community/wiki) in this repository
```

Comment:
Do we want to add "deprecated" here? Our discussion was to create *new* groups via the forum and move the old ones over in time, so lets try not to create new ones here?

Reply:
+1 for adding "will be deprecated"

Reply:
yeah, let's be clear;) done, PTAL